task2.js

```diff
@@ -1,31 +1,32 @@
 import fs from 'fs';
+import readline from 'readline';
 import csvtojson from "csvtojson";
 
 const task2 = () => {
-    const csvFile = './csv/1.csv';
+    const csvFile = './csv/nodejs-hw1-ex1.csv';
+    const txtFile = './txt/nodejs-hw1-ex2.txt';
 
     const readable = fs.createReadStream(csvFile);
-    const writable = fs.createWriteStream('./txt/out.json');
-
-
-    // readable
-    // .pipe(csvtojson())
-    // .on('data', chunk => console.log(chunk))
-    // .pipe(writable);
+    const writable = fs.createWriteStream(txtFile);
 
+    const rl = readline.createInterface({
+        input: csvtojson().fromStream(readable),
+        output: writable,
+        terminal: false
+    });
 
-    csvtojson().fromStream(readable).pipe(writable);
+    rl.on('line', line => rl.output.write(line))    
```

Comment:
Everything is great now! But, It would be better if you add '\n' after each line. Currently, the output file is a big line